operations/aws-security-roles/eis-security-audit-trusting-role.yml

```diff
@@ -2,7 +2,8 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: Mozilla Enterprise Information Security (EIS) Security Audit Role
 Metadata:
   Source: https://github.com/mozilla/security/tree/master/operations/aws-security-roles/eis-security-audit-trusting-role
-  TemplateVersion: 3.0.0
+  URL: https://s3.us-west-2.amazonaws.com/public.us-west-2.infosec.mozilla.org/infosec-security-roles/cf/eis-security-audit-trusting-role.yml
```

Comment:
I liked that it was referencing a VCS URL, the S3 bucket object doesn't have any sense of versioning.